Gemfile.lock

```diff
@@ -95,9 +95,9 @@ GEM
     grape-entity (0.5.1)
       activesupport
       multi_json (>= 1.3.2)
-    grape-swagger (0.23.0)
+    grape-swagger (0.25.3)
```

Comment:
Oh, Sorry. My mistake, i will. 
hmm seems the pull request is not updating. So will close this PR and send a new one instead. That should work